src/adapter/sources.ts

```diff
@@ -373,10 +374,19 @@ export class SourceContainer {
       const map = this._sourceMaps.get(compiled._sourceMapUrl!)!.map;
       if (!map)
         continue;
-      const entry = map.generatedPositionFor({source: sourceUrl, line: uiLocation.lineNumber, column: uiLocation.columnNumber});
+      const entry = map.generatedPositionFor({
+        source: sourceUrl,
+        line: uiLocation.lineNumber,
+        column: uiLocation.columnNumber - 1, // source map columns are 0-indexed
```

Comment:
I think technically speaking, we should be checking the `linesStartAt1` and `columnsStartAt1` properties of the initialize args and adjusting appropriately, but I'm not sure if or when that might ever be false.

Reply:
Not sure either. We assume 1-indexing elsewhere quite heavily. We should probably adjust this for correctness, https://github.com/microsoft/vscode-pwa/issues/88